docs/data-structures/tower-of-hanoi.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Tower of Hanoi
+slug: /data-structures/tower-of-hanoi
+---
+
+## Introduction
+
+In this we are given three towers. Tower 1 has n disks placed one over the other in ascending order such that the smallest disk is at the top of Tower 1 similar to cone shape, where n is any positive number. Tower 2 and Tower 3 are empty. We have to move all discs from Tower 1 to Tower 3 using Tower 2.
+
+## Explanation 
+
+<img src="https://i.imgur.com/PCk9mpI.jpg?1" width="900" height="250"/>
```

Comment:
Make sure all the images used are your own creation.

Reply:
Yes sir its created by me